### PR TITLE
Fetch data From the Space-X API

### PR DESCRIPTION
**Pull request by kennankole**
* Retrieve Data from the API
* Rendered the `Missions` data on the UI
* Created the `Missions` page. 
* Created `Status` component

### CODE REVIEW

**Review by Mujeeb4582 — CHANGES_REQUESTED** [review]
Good

File: src/routes/Missions/description.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prop-types */
+import Status from './status';
```
Comment (Mujeeb4582) [inline src/routes/Missions/description.js:3]: - [ ] Kindly enable the eslint for good practice.
- [ ] Solution for removing that error you follow the following steps
1. run it in terminal `npm i -S prop-types'
2. then include `import PropTypes from 'prop-types';` into your js file
3. at the end of the code you can include this
 
` Rocket.propTypes = {
  rocket: PropTypes.shape({
    rocketName: PropTypes.string.isRequired,
    flickerImages: PropTypes.string.isRequired,
    description: PropTypes.string.isRequired,
  }).isRequired,
}; `

**Review by Mujeeb4582 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @kennankole 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/routes/Missions/description.js (hunk 1 above)
Comment (kennankole) [reply src/routes/Missions/description.js]: Thanks @Mujeeb4582 
Let me implement these changes

**Review by Mujeeb4582 — APPROVED** [review]
Hi @kennankole  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Gitflow
✅ No linter error


## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**